katalog/thanos/README.md

```diff
@@ -0,0 +1,95 @@
+# Thanos
+
+Basically Thanos is an opensource Prometheus setup that allows to have 2 important features:
+
+1. high availability on Prometheus (setting more prometheus replicas)
+2. long term storage capacity relying on an external object storage
+
+This is the simplified diagram of a Thanos architecture:
+
+![thanos-architecture](images/thanos-arch.png)
+
```

Comment:
Can you add here the versions of the Thanos components?

Reply:
can you explain better this point? in the diagram or just in the README.md as matrix table?

Reply:
see https://github.com/sighupio/fury-kubernetes-kong/blob/master/katalog/kong/README.md image repository and tag section